index.js

```diff
@@ -21,14 +19,25 @@ exports.register = function(server, options, next) {
       })
     }, (err, resp, payload) => {
       if (err) {
-        sever.log(['micro-metrics', 'error'], err);
+        server.log(['micro-metrics', 'error'], err);
         return;
       }
       if (options.verbose) {
         server.log(['micro-metrics', 'track'], { type, tags, value, fields });
       }
     });
   });
+  if (options.logTrack) {
+    server.on('log', (event, tags) => {
+      options.logTrack.forEach((logTrack) => {
+        Object.keys(tags).forEach((tag) => {
+          if (tag === logTrack.logTag) {
+            server.track(logTrack.metricType, logTrack.value, logTrack.metricTags, {});
```

Comment:
we need to default value to 1 if its not set.  and what happens if no metricTags are set, does it still work?